tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/GradeImportConfirmationStep.java

```diff
@@ -21,14 +23,14 @@
  */
 public class GradeImportConfirmationStep extends Panel {
 
-    private static final Logger log = Logger.getLogger(GradeItemImportSelectionStep.class);
+    private static final Logger LOG = Logger.getLogger(GradeImportConfirmationStep.class);
 
     @SpringBean(name="org.sakaiproject.gradebookng.business.GradebookNgBusinessService")
     protected GradebookNgBusinessService businessService;
 
     private String panelId;
 
-    public GradeImportConfirmationStep(String id, List<ProcessedGradeItem> itemsToCreate, List<ProcessedGradeItem> itemsToUpdate,
+    public GradeImportConfirmationStep(String id, final List<ProcessedGradeItem> itemsToCreate, final List<ProcessedGradeItem> itemsToUpdate,
```

Comment:
At some point we need to swap the direct data passing in the constructors for an IModel that wraps it. Could have page specific models